tutorials/market/PyData_Demo_2.ipynb

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
<p>same thing about reset_index and copy</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Will fix this in the next commit.</p>